examples/group_import.rs

```diff
@@ -20,11 +20,10 @@ struct GAPGroup {
 
 impl GAPGroup {
     fn to_group(self) -> Group {
-        Group::from_iter(
-            self.generators
-                .into_iter()
-                .map(|images| ClassicalPermutation::from_slice(&images[..]).into()),
-        )
+        self.generators
```

Comment:
With this we can remove the FromIter import at the top

Reply:
I've merged the changes from #86 , so these should be resolved now